node-rti/src/ext.rs

```diff
@@ -51,6 +52,40 @@ impl ProgramStorage for ExtProgramStorage {
     }
 }
 
+impl ExtProgramStorage {
+    pub fn iter(&self) -> ExtProgramStorageIter {
```

Comment:
add test for this that it actually iterates after submitting several programs

Reply:
Done. Had to add a couple of dev-dependencies to `Cargo.toml`, but I think they will be necessary for testing other parts of this module anyway